Formula/d/dvd+rw-tools.rb

```diff
@@ -1,12 +1,12 @@
 class DvdxrwTools < Formula
   desc "DVD+-RW/R tools"
-  homepage "http://fy.chalmers.se/~appro/linux/DVD+RW/"
-  url "http://fy.chalmers.se/~appro/linux/DVD+RW/tools/dvd+rw-tools-7.1.tar.gz"
+  homepage "https://fy.chalmers.se/~appro/linux/DVD+RW/"
+  url "https://fy.chalmers.se/~appro/linux/DVD+RW/tools/dvd+rw-tools-7.1.tar.gz"
```

Comment:
we might need some mirror for this, I could not download this url, hanging for me.

Reply:
Works for me, works for CI, maybe a fluke?